app/assets/javascripts/components/gtm-form-listener.js

```diff
@@ -0,0 +1,35 @@
+function GTMFormListener ($form, $dataLayer) {
+  this.$form = $form
+  this.dataLayer = $dataLayer || window.dataLayer
+}
+
+GTMFormListener.prototype.handleSubmit = function (event) {
+  var eventName = this.$form.dataset.gtm
+  var message = {}
```

Comment:
Does this need to be set to an empty object if we do that inside the element loop?

Reply:
No, fixed in 900499c48755f2913ce8bf958f3ae6e520c9b905